docs/src/tutorials/algorithms/tsp_lazy_constraints

```diff
@@ -0,0 +1,286 @@
+
+# Copyright (c) 2019 Arpit Bhatia and contributors                               #src
```

Comment:
your name here

Reply:
I'm sorry but where exactly? I copied the copyright notice from the root of the project.